Analyse_Birds.qmd

```diff
@@ -107,13 +122,220 @@ required_packages <- c(
   "lattice",    # for multipanel graphics
   "dplyr",      # for data manipulation
   "FactoMineR", # for ordinations
-  "factoextra"
+  "factoextra", 
+  "sf",         # for coordinats manipulation
+  "terra",      # for maps creation
+  "spData", 
+  "tmap", 
+  "leaflet", 
+  "spDataLarge", 
+  "maps", 
+  "gifski"      # for gifs display
 )
 
 # Install and load the packages
 install_if_missing(required_packages)
 ```
 
+# First look at the data : 
+
+You can visualise the first rows of the dataset by running the following cell :
+```{r}
+# View the firsts lignes of the dataset
+head(data)
+```
+
+In order to picture the data in a more visual way, we will be representing our dataset on a map. 
```

Comment:
"We will represent" is better english

Reply:
Modification done, thank you for the review